core/sodasql/cli/cli.py

```diff
@@ -514,24 +517,24 @@ def scan(scan_yml_file: str, warehouse_yml_file: str, variables: tuple, time: st
     type=Path,
 )
 @click.option(
-    "--dbt-manifest",
+    "--manifest-file",
```

Comment:
does click do autocomplete? if so, maybe it's ok if not I would air on the side of shorter option names (which I know is not ideal from a clear internal naming point of view but I wouldn't be surprised if click had a way to internally map something to something else. I think I've done it with argparse before.

Reply:
with shorter you mean `--dbt-manifest` to `--manifest` without the `-file` suffix?

Reply:
Yeah maybe? What do you think? I like the specificity but I also am lazy. I leave it up to you! we can always add short versions of those if users complain.